examples/cairo_threads/image/image_holder.rs

```diff
@@ -0,0 +1,42 @@
+use std::cell::RefCell;
+use std::rc::Rc;
+/// Helper struct that allows passing the pixels to the Cairo image surface and once the
```

Comment:
Please add an empty line before the doc comment.

Reply:
Mmmh, too bad that rustfmt does not do that for me

Reply:
Open an issue there? :)

Reply:
Done (the line change not the issue)

Reply:
Please open the issue. Everyone will benefit from it.

Reply:
Sure, will see first if it exists already

Reply:
Thanks!